src/styles/index.scss

```diff
@@ -1,3 +1,46 @@
 body {
   margin: 0;
 }
+
+.card {
+  max-width: 200px;
+
+  background-color: bisque;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__picture {
+  max-width: 160px;
+}
+
+.card__item--model {
+  font-family: Roboto, sans-serif;
```

Comment:
you need to use @font-face or link it into head (but only wanted variants) 
![image](https://github.com/user-attachments/assets/c558e3cb-77f9-4b71-b8ac-4f8e1f9376e3)